scripts/just.config.ts

```diff
@@ -95,7 +97,7 @@ module.exports = function preset() {
 
   task('lint', parallel('lint-imports', 'tslint'));
```

Comment:
will eslint be added here eventually?

Reply:
Yes, it will replace tslint on the final PR--I just didn't do that yet because it would cause tons of lint errors without the corresponding updates to local disables and such.